packages/web/src/pages/signup/SignupPage.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from "react";
+import { useMutation } from "react-apollo";
+import { gql } from "apollo-boost";
+import { Formik, Field, Form, FormikActions } from "formik";
+import styled from "styled-components";
+import { Input } from "../../components/Input";
+import { Button } from "../../components/Button";
+import { theme } from "../../config";
+import { useHistory } from "react-router";
+
+interface FormValues {
+  emailAddress: string;
+  password: string;
+}
+
+const Container = styled.div`
+  max-width: 400px;
+  margin-right: auto;
+  margin-left: auto;
+  box-shadow: 0px 0px 10px lightgrey;
+  padding: 30px 50px 50px;
+  margin-top: 10vh;
+`;
+
+const Title = styled.h1`
+  margin: 20px 0;
+  font-size: 20px;
+  font-weight: bold;
+`;
+
+const StyledForm = styled(Form)`
+  input {
+    margin-bottom: 20px;
+  }
+  button {
+    margin-top: 20px;
+  }
+`;
+
+const StyledErrorMessage = styled.div`
+  color: ${theme.colors.negative};
+`;
+
+export const SignupPage = () => {
+  const history = useHistory();
+  const [signUpMutation, { loading, error }] = useMutation(
+    gql`
+      mutation signUp($input: UserAuthInput!) {
+        signUp(input: $input) {
+          id
+          emailAddress
+          jwt
+        }
+      }
+    `,
+    {
+      onCompleted({ signUp }) {
+        localStorage.setItem("token", signUp.jwt);
+        history.push("/");
+      }
+    }
+  );
```

Comment:
Use apollo hooks to send a mutation to the server. `signUpMutation` will be used when submitting the form.